src/Components/PaymentMethod/PaymentMethodContainer.js

```diff
@@ -372,6 +682,84 @@ const PaymentMethodContainerWithoutStripe = ({
         }
       });
     }
+
+    if (cardProcessor === "tap" && !selectedPaymentMethodId) {
+      const tapKey = Tapjsli(
+        window.Pelcro.site.read()?.tap_gateway_settings
+          .publishable_key
+      );
+
+      let elements = tapKey.elements({});
+
+      let style = {
+        base: {
+          color: "#535353",
+          lineHeight: "18px",
+          fontFamily: "sans-serif",
+          fontSmoothing: "antialiased",
+          fontSize: "16px",
+          "::placeholder": {
+            color: "rgba(0, 0, 0, 0.26)",
+            fontSize: "15px"
+          }
+        },
+        invalid: {
+          color: "red"
+        }
+      };
+
+      // input labels/placeholders
+      let labels = {
+        cardNumber: "Card Number",
+        expirationDate: "MM/YY",
+        cvv: "CVV",
+        cardHolder: "Card Holder Name"
+      };
+
+      //payment options
+      let paymentOptions = {
+        labels: labels,
+        TextDirection: "ltr"
+      };
+
+      //create element, pass style and payment options
+      let card = elements.create(
+        "card",
+        { style: style },
+        paymentOptions
+      );
+
+      //mount element
+      card.mount("#tapPaymentIframe");
+
+      //card change event listener
+      card.addEventListener("change", function (event) {
+        // if (event.error_interactive) {
```

Comment:
is this code needed anytime?

Reply:
Yes, for error handling, we didn't decide on how we will do it just yet.